WindowsServerDocs/virtualization/hyper-v/manage/Manage-Hyper-V-integration-services.md

```diff
@@ -307,4 +310,7 @@ For hosts earlier than Windows Server 2016 and Windows 10, you'll need to manual
   
 4.  After the installation finishes, all integration services are available for use.
 
-These steps can't be automated or done within a Windows PowerShell session for online virtual machines. You can apply them to offline VHDX images; [see this blog post](https://blogs.technet.microsoft.com/virtualization/2013/04/18/how-to-install-integration-services-when-the-virtual-machine-is-not-running/).
+> [!NOTE]
+> These steps **can't be automated** or done within a Windows PowerShell session for **online** virtual machines.
+> You can apply them to **offline** VHDX images; [see this article](https://docs.microsoft.com/en-us/virtualization/community/team-blog/2013/20130418-how-to-install-integration-services-when-the-virtual-machine-is-not-running).
+> You can also automate the deploy of the integration services through **Configuration Manager** with the VMs **Online**, but you need to restart the VMs at the end of the installation; [see this article](https://docs.microsoft.com/en-us/archive/blogs/manageabilityguys/deploying-hyper-v-integration-services-to-vms-using-config-manager-and-dism)
```

Comment:
```suggestion
> You can also automate the deploy of the integration services through **Configuration Manager** with the VMs **Online**, but you need to restart the VMs at the end of the installation; see [Deploying Hyper-V Integration Services to VMs using Config Manager and DISM](https://docs.microsoft.com/archive/blogs/manageabilityguys/deploying-hyper-v-integration-services-to-vms-using-config-manager-and-dism)
```
Please also note: if you disagree with parts of my suggestions, you are free to edit the commit suggestion during the commit process (or you can make the necessary changes yourself to get the desired result).

Reply:
Many tnx for the suggestion ;)